docs/developers/v2/hacking-with-yearn.md

```diff
@@ -2,148 +2,156 @@
+A Yearn Strategy is a set of smart contracts that implement different farming strategies that utilize the deposited assets to generate the best yields for users.
 
 The implementation of the strategies moves assets from one farming application to another one with higher interest. This will mostly be done automatically, which automatically monitors and tracks the best farming applications with high yield and low risks.
 
-Rewards/profits generated by strategies are regularly updated, harvested, swapped for the original vault asset, and deposited again for compound farming. Each vault has a unique strategy and the strategy will be improved constantly based on the observations of our AI-based monitoring engines. 
+Rewards/profits generated by strategies are regularly updated, harvested, swapped for the original vault asset, and deposited again for compound farming. Each vault has a unique strategy and the strategy will be improved constantly based on the observations of our AI-based monitoring engines.
 
-By having a strategy, users only need to deposit single assets and let the strategies generate profits from the funds. Yearn automates the whole compounding process, making it as close to optimal as possible. 
+By having a strategy, users only need to deposit single assets and let the strategies generate profits from the funds. Yearn automates the whole compounding process, making it as close to optimal as possible.
 
-* https://github.com/yearn/brownie-strategy-mix
+- <https://github.com/yearn/brownie-strategy-mix>
 
-* Forking a Strategy Locally: https://github.com/yearn/yearn-vaults/blob/main/CONTRIBUTING.md
+- Forking a Strategy Locally: <https://github.com/yearn/yearn-vaults/blob/main/CONTRIBUTING.md>
 
-# Data Analysis
+## Data Analysis
 
-Yearn aims to provide more insights into our products to our customers and partners through analyzing the on-chain and off-chain data. Given that all Yearn protocol data is publicly stored on-chain, there are lots of insights to be gained from analyzing these transactions. 
+Yearn aims to provide more insights into our products to our customers and partners through analyzing the on-chain and off-chain data. Given that all Yearn protocol data is publicly stored on-chain, there are lots of insights to be gained from analyzing these transactions.
 
 Two common tools for on-chain data analysis are:
+
 1. **Dune Analytics**- Query contract data and generate dashboards with SQL
+
 - [Current Dashboards](https://dune.com/projects/yearn )
-    
     Here's a list of all the new things you can query for:
-    - harvests
-    - deployments
-    - transactions
-    - contract_tokens
-    - contract_strategy
-    - v2_deployments
+  - harvests
+  - deployments
+  - transactions
+  - contract_tokens
+  - contract_strategy
+  - v2_deployments
 
-Fiddle with them all at https://dune.com/queries/625814
+  Fiddle with them all at <https://dune.com/queries/625814>
```

Comment:
```suggestion
  Fiddle with them all at https://dune.com/queries/625814
```

Reply:
seems like all of these dashboards are either broken or use the old version of Dune and need to be migrated.